README.md

```diff
@@ -2,7 +2,7 @@
 
 A combination of circuit breaker and timeout. The .NET version of the open source [Hystrix library](https://github.com/Netflix/Hystrix) built by Netflix.
 
-[![Build Status .NET](https://ci.appveyor.com/api/projects/status/github/Travix-International/Hystrix.Dotnet?svg=true)](https://ci.appveyor.com/project/Travix-International/Hystrix.Dotnet/)
+[![Build Status .NET](https://ci.appveyor.com/api/projects/status/1a09t29hcvfo1bbq/branch/master?svg=true)](https://ci.appveyor.com/project/Travix-International/Hystrix.Dotnet/)
```

Comment:
Did appveyor change their status urls?

Reply:
No, but we were using a wrong link, which would show the status of the latest build *regardless of the branch*. So if a build in a PR branch failed, that made the badge red on the home page.

Although I don't know why the structure of the badge route is different (it looks kinda obfuscated now). I just went to the AppVeyor "Badges" page and grabbed the link from there. Maybe they support both, and now this is the default.